src/Test.EndToEnd.Baselining/Extensions/SarifLogExtensions.cs

```diff
@@ -59,7 +59,9 @@ public static IEnumerable<Run> EnumerateRuns(this SarifLog log)
 
         public static Result FindByGuid(this SarifLog log, string guid)
```

Comment:
>FindByGuid [](http://example.com/codeflow?start=29&length=10)

Should we simply have updated this API to accept a GUID argument?? Something to keep in mind moving forward.